var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -169,7 +171,8 @@ def install(self, spec, prefix):
             '--with-psm2' if '+psm2' in spec else '--without-psm2',
             '--with-mxm' if '+mxm' in spec else '--without-mxm',
             # Other options
-            '--enable-mpi-thread-multiple' if '+thread_multiple' in spec else '--disable-mpi-thread-multiple',  # NOQA: ignore=E501
+            ('--enable-mpi-thread-multiple' if '+thread_multiple' in spec
+                 else '--disable-mpi-thread-multiple'),
```

Comment:
I used the indentation of the parenthesis, and then four more spaces. I don't care much about the details, but I wanted to avoid the overlong line. Do you want me to remove one space?